### ISSUE TITLE
Feature request: Make it easier to bootstrap a new project with dip install

### ISSUE THREAD

**Issue by blijblijblij** [bibendi/dip#94]
**Describe the bug**
Not really a bug, more of a feature request

Allow dip to `dip install` the newest files needed for dip to run and manage itself:

A simpler way of kicking of dip in a new project as I caught myself setting up a new project via:
- `cp -r ../some-other-project/.dockerdev .`
- `cp ../some-other-project/dip.yml .`
- `cp ../some-other-project/docker-compose.yml .`

**To Reproduce**
1. Provide dip.yml config
2. Provide docker-compose.yml
3. Run command

**Expected behavior**
`dip install`

gives me 

- `.dockerdev` folder
- `docker-compose.yml` the latest and greatest version
- `dip.yml` the latest and greatest

**Screenshots**

**Context (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04.3]
 - Version [e.g. binary 4.3.2]

**Additional context**
I agree this has some RTFM to it, as I would be able to start from the latest and greatest by searching around a little, but still, I think this would make for a great extension of an already great 🚀 gem.

**Comment by bibendi** [comment]
Hi!
I've started doing something like that a year ago, but then I realized that every single project has a unique configuration.

https://github.com/bibendi/dip/pull/66

**Comment by blijblijblij** [comment]
> Hi!
> I've started doing something like that a year ago, but then I realized that every single project has a unique configuration.
> 
> #66

Cool, and yeah agree on the uniqueness of a configuration, but this #66 still seems like a sane default.

So that at least one can hit the ground running after a `rails new`, `gem install dip` and a `dip generate`.

After that you will always start adapting, but this way you'll kickstart users with the basics...

**Comment by bibendi** [comment]
Agree. I'll try to finish that if I have more free time this fall.
